rsconf/component/mpi_worker.py

```diff
@@ -13,6 +13,11 @@
 
 
 class T(component.T):
+    def gen_host_and_identity_ssh_keys(self, jc, z):
+        return super().gen_host_and_identity_ssh_keys(
+            jc, self.name + "/" + z.user, visibility="channel"
```

Comment:
z.user could be assigned self.user_name, or devbox could put user in z. The goal is to share as much code with gen_host_and_identity_ssh_keys. I don't think this needs to be a subclassed method.

Reply:
How does this look ccdb931 ?